src/orbit/MaterialInteractions/Collimator.cc

```diff
@@ -347,24 +347,26 @@ int Collimator::checkCollFlag(double x, double y){
 	
 
 int Collimator::driftParticle(int coll_flag, double& zrl, double length, double* coords, SyncPart* syncpart){
+	double eps = 1.0e-8;
+	double dlength = length * 1.0e-4;
+	double stepsize = 0.001;
+
 	while((coll_flag == 0) && (zrl > 0))
 	{
 		
-		double x = coords[0];
-		double xp = coords[1];
-		double y = coords[2];
-		double yp = coords[3];
-		double dlength = length * 1.0e-4;
-		double stepsize = 0.001;
+		//double x = coords[0];
```

Comment:
Maybe we better remove these lines completely? Having them as comments doesn't seem relevant.